apps/web/src/pages/PoolCandidates/ViewPoolCandidatePage/components/ApplicationPrintButton/ApplicationPrintDocument.tsx

```diff
@@ -0,0 +1,891 @@
+                  "Document title for printing a user's application snapshot.",
+              })}
+            </Heading>
+            <Heading level="h2" data-h2-font-weight="base(700)">
+              <>{getFullNameLabel(user.firstName, user.lastName, intl)}</>
+            </Heading>
+            {relevantPoolCandidate && (
+              <>
+                <PageSection>
+                  <Heading level="h3" data-h2-font-weight="base(700)">
+                    {intl.formatMessage({
+                      defaultMessage: "Education requirement",
+                      id: "/zx1kX",
+                      description: "Education requirement section header.",
+                    })}
+                  </Heading>
+                  <p>
+                    {intl.formatMessage({
+                      defaultMessage: "Requirement selection",
+                      id: "bp1VHg",
+                      description: "Label before required selection listed.",
+                    })}
+                    {intl.formatMessage(commonMessages.dividingColon)}
+                    {relevantPoolCandidate.educationRequirementOption
+                      ? intl.formatMessage(
+                          getEducationRequirementOption(
+                            relevantPoolCandidate.educationRequirementOption,
+                          ),
+                        )
+                      : intl.formatMessage(commonMessages.notAvailable)}
+                  </p>
+                </PageSection>
+                <PageSection>
+                  <p>
+                    {intl.formatMessage({
+                      defaultMessage:
+                        "Demonstrated with the following experiences",
+                      id: "1CRt78",
+                      description:
+                        "label before listing experience associated with fulfilling requirements.",
+                    })}
+                  </p>
+                  {educationRequirementExperiencesFiltered.length > 0 && (
+                    <ul>
+                      {educationRequirementExperiencesFiltered.map(
+                        (experience) => {
+                          return (
+                            <EducationRequirementExperience
+                              experience={experience}
+                              key={`${experience.id}-education-requirement`}
+                            />
+                          );
+                        },
+                      )}
+                    </ul>
+                  )}
+                </PageSection>
+                <Heading level="h3" data-h2-font-weight="base(700)">
+                  {intl.formatMessage({
+                    defaultMessage: "Essential skills",
+                    id: "w7E0He",
+                    description:
+                      "Title for the required skills snapshot section",
+                  })}
+                </Heading>
+                <BreakingPageSection>
```

Comment:
I think we want this to wrap the heading as well, otherwise there is a big gap between the heading and the list of skills.

Reply:
Sure can do https://github.com/GCTC-NTGC/gc-digital-talent/pull/8185/commits/32a0fbbadc5dcc622b170e7d7c4984f79b4b8f78